### PR TITLE
Added Traffic RPC.

### PR DESCRIPTION
**Pull request by melvinlu**
Added traffic RPC that utilizes Bing's traffic API. Included is an example & custom blocks. You will need a Bing traffic API key.

There is not much functionality to this RPC. Currently it just returns an array populated with coordinates of traffic incidents in a general area.

### CODE REVIEW

**Comment by brollb** [comment]
Traffic type should probably be decoded on the server. That is, `type` should be "accident", "construction", etc, not 1, 2, etc:
traffictype

Also, for severity, I think we should either return the severity for each accident or (probably the better choice) just remove it altogether and return them all. It isn't really clear what the scale is and it might be more cumbersome to explain it here (on the block).
trafficseverity

**Comment by melvinlu** [comment]
Thanks. I went ahead and just removed the severity parameter (turns out I never actually implemented it with this commit).

**Comment by brollb** [comment]
I just realized that the traffic messages don't stop when the map changes and clicking the map resets the map and starts the messages... I would probably make clicking only start the traffic messages and have the traffic messages stop and clear on map updates

**Comment by melvinlu** [comment]
I'm not sure if  I understood you correctly, can you take a look at the change? I had the messages stop and clear on map updates. However, there is a limit on how "wide" the zoom can be for the API so I just force the map to be at zoom 5 whenever the messages come in.

**Comment by brollb** [comment]
also, what happens if the area is too large? does it return nothing or just an incomplete set?

**Comment by melvinlu** [comment]
How's that? (Changed `trigger` to be a reporter block that will return an error if the area is too large, and added the four navigation blocks) 

I also went ahead and added back in the zoom out block since the `trigger` block will now return an error.

File: src/server/rpc/libs/RPC [hunk 1]
```diff
@@ -8,3 +8,4 @@ weather.xml	Weather API
 pubsub.xml	PubSub
 kvstore.xml	Key-Value Store
 battleship.xml	Battleship
+Traffic.xml	Traffic
```
Comment (brollb) [inline src/server/rpc/libs/RPC]: change `Traffic.xml` to `traffic.xml` - this will break on linux which is case sensitive

**Comment by brollb** [comment]
One last fix and it should be good!

**Comment by melvinlu** [comment]
Ah! My bad! There you go.

**Comment by brollb** [comment]
awesome - thanks!

**Pull request merged**